components/02-elements/skip/_skip.scss

```diff
@@ -0,0 +1,7 @@
+.skip {
```

Comment:
it's hidden link supported accessibility issue it serves like a landmark or bypass link, I want to create more links like that, so we can change the name, but this link serves to skip some content. I can change the name to `hidden-link` ?